test/unit/shared/access_permissions_test_base.rb

```diff
@@ -5,15 +5,19 @@ module AccessPermissionsTestBase
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def check_routes(app_routes, skipped_actions)
+    def should_skip_path?(path, skipped_actions = [], skip_patterns = [])
```

Comment:
`skip_actions` and `skip_patterns` are always passed, no point in the defaults here (defaults are only used if you don't pass a parameter, if a param is passed it takes precedent even if it's `nil`)

Reply:
Yeah, I overengineered it a bit, as it has only one user as of now, no reason for defaults :)